docs/projects/ad469x_fmc/index.rst

```diff
@@ -0,0 +1,242 @@
+.. _ad469x_fmc:
+
+AD469X-FMC HDL project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+The AD469X HDL reference design provides all the interfaces that are
+necessary to interact with the devices on the :adi:`EVAL-AD4696 <EVAL-AD4696>`
+board.
+
+The design has a SPI Engine instance to control and acquire data from the
+:adi:`AD4696` 16-bit precisions ADC, providing support to capture continuous
+samples at maximum sampling rate. Currently the design supports the Zedboard.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD4696 <EVAL-AD4696>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD4695`
+-  :adi:`AD4696`
+-  :adi:`AD4697`
+-  :adi:`AD4698`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+
+Block design
+-------------------------------------------------------------------------------
+
+The reference design uses the standard :ref:`SPI Engine Framework <spi_engine>`
+to interface the :adi:`AD4696` ADC in single SDO Mode. The :ref:`SPI Engine
+Offload module <spi_engine offload>`, which can be used to capture continuous
+data stream at maximum data rate, is triggered by the BUSY signal of the device. 
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+AD469X_FMC
```

Comment:
Could you remove this section? It confuses me like another block diagram would follow, for another eval board.

Reply:
one.